lifecycle/import.go

```diff
@@ -0,0 +1,49 @@
+package lifecycle
+
+import (
+	"github.com/jfrog/jfrog-cli-core/v2/utils/config"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+type ReleaseBundleImportCommand struct {
+	releaseBundleCmd
+	filePath string
+}
+
+func (rbi *ReleaseBundleImportCommand) ServerDetails() (*config.ServerDetails, error) {
+	return rbi.serverDetails, nil
+}
+
+func (rbi *ReleaseBundleImportCommand) CommandName() string {
+	return "rb_export"
+}
+
+func NewReleaseBundleImportCommand() *ReleaseBundleImportCommand {
+	return &ReleaseBundleImportCommand{}
+}
+func (rbi *ReleaseBundleImportCommand) SetServerDetails(serverDetails *config.ServerDetails) *ReleaseBundleImportCommand {
+	rbi.serverDetails = serverDetails
+	rbi.releaseBundleCmd.serverDetails = serverDetails
+	return rbi
+}
+
+func (rbi *ReleaseBundleImportCommand) SetFilepath(filePath string) *ReleaseBundleImportCommand {
+	rbi.filePath = filePath
+	return rbi
+}
+
+func (rbi *ReleaseBundleImportCommand) Run() (err error) {
+	if err = validateArtifactoryVersionSupported(rbi.serverDetails); err != nil {
```

Comment:
i've added a function that will check if it is cloud platform, but what do you suggest about the version constraint? there is guarantee that it will be fixed the next patch.

Reply:
The problem with checking whether the instance is a cloud is that it requires admin privileges, which is not a constraint we would like to set.
To be on the safe side, I would suggest to keep the command hidden till the patch is officially out, then exposing it with the same minimal version for both cloud and self hosted.